### ISSUE TITLE
ENH: Add support for media

### ISSUE THREAD

**Issue by MashyBasker** [Cloud-Code-AI/kaizen#415]
Enhance our PR description and issue label generators to support multimedia inputs (screenshots, videos, GIFs) in addition to text, for more comprehensive content analysis.

**Comment by SwapnilChand** [comment]
Are we using any vision models as of now?

**Comment by MashyBasker** [comment]
I don't think so. Maybe we could use [LLaVA](https://huggingface.co/liuhaotian/llava-v1.5-7b)?
